glam/api/models.py

```diff
@@ -267,3 +267,19 @@ class Meta:
                 fields=["channel", "build_id"],
             )
         ]
+
+
+class UsageInstrumentation(models.Model):
+    ACTION_PROBE_SEARCH = 'PROBE_SEARCH'
+    ACTIONS = [
+        (ACTION_PROBE_SEARCH, "probe_search"),
+    ]
+    id = models.AutoField(primary_key=True)
+    action_type = models.CharField(max_length=100, choices=ACTIONS)
+    timestamp = models.DateTimeField(auto_now_add=True)
+    context = models.JSONField(null=True)
+    tracking_id = models.CharField(null=True, max_length=500)
+    probe_name = models.CharField(null=True, max_length=100)
```

Comment:
`CharField`s are suggested to not allow a null since that leaves 2 values that are basically no data. Check out the docs for more on this: https://docs.djangoproject.com/en/4.0/ref/models/fields/#null

Reply:
Using blank for chars and text as per django's convention